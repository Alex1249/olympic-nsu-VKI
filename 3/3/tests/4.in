5 10
0.63