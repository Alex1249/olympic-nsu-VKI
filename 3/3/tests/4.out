0.3033333333
